.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,12 +5,11 @@ Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/bond/blob/devel
 I, the developer opening this PR, do solemnly pinky swear that:
 
 - [ ] I've followed [the instructions](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
-- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
-- [ ] I've updated the [FISMA documentation](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing
```

Comment:
fisma documentation was replaced with the security impact and clia risk fields on jira tickets